### PR TITLE
uchardet

### PR DESCRIPTION
**Pull request by scopatz**
new lib

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is ready for review! It is mac/linux only, and passes on those platforms

File: recipes/uchardet/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+mkdir -p build
+cd build
+cmake -DLIB_INSTALL_DIR="${PREFIX}/lib" \
+  -DCMAKE_BUILD_PREFIX="${PREFIX}" \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+  -DCMAKE_BUILD_TYPE=Release \
+  ..
+make -j "${CPU_COUNT}"
+make install
```
Comment (ocefpaf) [inline recipes/uchardet/build.sh]: Is there a `make check`?

File: recipes/uchardet/build.sh [hunk 2]
```diff
@@ -5,5 +5,4 @@ mkdir -p build
 cd build
 cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DCMAKE_BUILD_TYPE=Release ..
 make -j "${CPU_COUNT}"
-make check
```
Comment (ocefpaf) [inline recipes/uchardet/build.sh]: Oops. Ignore my comment.

File: recipes/uchardet/build.sh (hunk 1 above)
Comment (scopatz) [reply recipes/uchardet/build.sh]: There isn't.  I tried it and it failed, see https://github.com/conda-forge/staged-recipes/pull/8014/commits/6011f5399d8d66c4662ace13dd9d8e75f55e00fe

File: recipes/uchardet/build.sh (hunk 2 above)
Comment (scopatz) [reply recipes/uchardet/build.sh]: haha np

**Comment by scopatz** [comment]
Thanks @ocefpaf!

**Pull request merged**